src/_data/platforms.yml

```diff
@@ -1,18 +1,18 @@
 - platform: 'Android SDK'
   target-arch: 'x64, Arm32, Arm64'
-  supported: '21 to 35'
-  ci-tested: '21 to 35'
-  unsupported: '20 and earlier'
+  supported: '24 to 35'
```

Comment:
This can be updated to say 24 to 36 for both supported and tested

Reply:
Done. Thanks for calling this out!